### ISSUE TITLE
Stack Snippets do not load in the SE editor

### ISSUE THREAD

**Issue by ETHproductions** [vihanb/PPCG-Design#125]
When writing/editing a question/answer while the userscript is turned on, the Stack Snippet icon does not appear in the toolbar, nor do existing Stack Snippets render as they should (they just appear as code blocks). Snippets do render properly in answers that are not being edited. Everything works fine when the userscript is turned off.

I have a hunch that this is related to #124...

Tested in Firefox 54 on Windows 7

**Comment by kritixilithos** [comment]
I can confirm this issue for Chrome 59.

**Comment by vihanb** [comment]
1. obvious solution is to use [esfiddle](https://vihan.org/p/esfiddle) (cross-browser, ES7, syntax highlight, etc.) even supports array comprehension
 2. [no-repro]

**Comment by ETHproductions** [comment]
1. The whole point of stack snippets is to integrate the code right into the post, though I see your point...
2. You're on Mac, right? That might be part of the issue.

**Comment by vihanb** [comment]
@kritixilithos uses mac too though AFAIK

**Comment by kckennylau** [comment]
Windows 7 32-bit Chrome 59.0.3071.115 here:
1. Confirmed.
2. No repro.

**Comment by kritixilithos** [comment]
Yes, I also use macOS. @vihanb what browsers have you tested on?
